sdk/communication/azure-communication-rooms/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
The tag should also be updated from `current` to `dependency` to make this work. Otherwise, the build will fail for the wrong version tag.

Since you want to release this library with the dependency version of `azure-communication-common` the tag should be temporarily updated to `dependency`. After release, we should switch this back to `current`.
```suggestion
      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->
```